SalesforceSDK/Salesforce.SDK.Core/Source/Net/HttpCall.cs

```diff
@@ -89,15 +91,20 @@ public static string MimeType(this ContentTypeValues contentType)
     /// </summary>
     public class HttpCall
     {
-        private readonly ContentTypeValues ContentType;
-        private readonly HttpCallHeaders Headers;
-        private readonly HttpMethod Method;
-        private readonly string RequestBody;
-        private readonly string Url;
-        private readonly HttpClient WebClient;
-        private Exception HttpCallErrorException;
-        private string ResponseBodyText;
-        private HttpStatusCode StatusCodeValue;
+        /// <summary>
+        /// Use this property to retrieve the user agent.
+        /// </summary>
+        public static string UserAgentHeader { private set; get; }
+        private const string UserAgentHeaderFormat = "SalesforceMobileSDK/3.1 {0}";
```

Comment:
Added application name and application version to the UserAgent.

Example UserAgent:

SalesforceMobileSDK/3.1 (Salesforce.SDK.Store.Test/1.1.0 native) 5.0 (Windows NT 6.3; WOW64; Trident/7.0; Touch; .NET4.0E; .NET4.0C; Tablet PC 2.0; .NET CLR 3.5.30729; .NET CLR 2.0.50727; .NET CLR 3.0.30729; WebView/2.0; rv:11.0) like Gecko